src/plugins/discover/public/application/components/no_results/no_results.tsx

```diff
@@ -28,184 +28,155 @@
+  //       description: <EuiCode>status:[400 TO 499] AND (extension:php OR extension:html)</EuiCode>,
+  //       title: (
+  //         <EuiText size="s">
+  //           <strong>
+  //             <FormattedMessage
+  //               id="discover.noResults.searchExamples.400to499StatusCodeWithPhpOrHtmlExtensionExampleTitle"
+  //               defaultMessage="Find status codes 400-499 with the extension php or html"
+  //             />
+  //           </strong>
+  //         </EuiText>
+  //       ),
+  //     },
+  //   ];
+  //
+  //   luceneQueryMessage = (
+  //     <Fragment>
+  //       <EuiSpacer size="xl" />
+  //
+  //       <EuiText size="s">
+  //         <h3>
+  //           <FormattedMessage
+  //             id="discover.noResults.searchExamples.refineYourQueryTitle"
+  //             defaultMessage="Refine your query"
+  //           />
+  //         </h3>
+  //
+  //         <p>
+  //           <FormattedMessage
+  //             id="discover.noResults.searchExamples.howTosearchForWebServerLogsDescription"
+  //             defaultMessage="The search bar at the top uses OpenSearch&rsquo;s support for Lucene {queryStringSyntaxLink}.
+  //               Here are some examples of how you can search for web server logs that have been parsed into a few fields."
+  //             values={{
+  //               queryStringSyntaxLink: (
+  //                 <EuiLink
+  //                   target="_blank"
+  //                   href={getServices().docLinks.links.opensearch.queryDSL.base}
+  //                 >
+  //                   <FormattedMessage
+  //                     id="discover.noResults.searchExamples.queryStringSyntaxLinkText"
+  //                     defaultMessage="Query String syntax"
+  //                   />
+  //                 </EuiLink>
+  //               ),
+  //             }}
+  //           />
+  //         </p>
+  //       </EuiText>
+  //
+  //       <EuiSpacer size="m" />
+  //
+  //       <EuiDescriptionList type="column" listItems={searchExamples} />
+  //
+  //       <EuiSpacer size="xl" />
+  //     </Fragment>
+  //   );
+  // }
 
   return (
     <I18nProvider>
       <EuiPanel hasBorder={false} hasShadow={false} color="transparent">
-        <EuiCallOut
+        <EuiEmptyPrompt
+          iconType="alert"
+          iconColor="default"
+          data-test-subj="discoverNoResults"
           title={
-            <FormattedMessage
-              id="discover.noResults.searchExamples.noResultsMatchSearchCriteriaTitle"
-              defaultMessage="No results match your search criteria"
-            />
+            <EuiText size="s">
+              <h2>No results</h2>
+            </EuiText>
+          }
+          body={
+            <EuiText size="s" data-test-subj="discoverNoResultsTimefilter">
+              <p>
+                Try selecting a different data source, expanding your time range or modifying the
+                query &amp; filters.
+              </p>
+            </EuiText>
```

Comment:
Is there a reason we're not localizing this text?

Reply:
No, just updated with localization